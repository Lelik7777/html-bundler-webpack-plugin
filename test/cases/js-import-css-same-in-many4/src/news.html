<!DOCTYPE html>
<html>
<head>
  <title>News</title>
  <link href="./common.css" rel="stylesheet">
  <link href="./app.css" rel="stylesheet">
</head>
<body>
  <h1>News</h1>
  <div class="common">.common</div>
  <div class="app">.app</div>
</body>
</html>