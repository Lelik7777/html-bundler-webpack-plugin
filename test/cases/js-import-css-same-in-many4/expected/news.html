<!DOCTYPE html>
<html>
<head>
  <title>News</title>
  <link href="css/common.bundle.css" rel="stylesheet">
  <link href="css/app.bundle.css" rel="stylesheet">
</head>
<body>
  <h1>News</h1>
  <div class="common">.common</div>
  <div class="app">.app</div>
</body>
</html>